Sources/NSTimer+ClosureKit.swift

```diff
@@ -24,8 +24,8 @@ final class BlockTimer: NSObject {
     class func scheduledTimer(withTimeInterval interval: TimeInterval, repeated: Bool = false,
         handler: @escaping CKTimerHandler) -> Timer
     {
-        return Timer.scheduledTimer(timeInterval: interval, target: self,
```

Comment:
I don't think calling a method without the fully qualified class helps in terms of readability when you are looking at this in isolation. What's your reasoning for this change (other than 'it compiles without it' 😉 )?